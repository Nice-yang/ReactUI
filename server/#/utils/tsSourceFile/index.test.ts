import * as ts from 'typescript';
import { createSourceFile$ } from './index';

it('reads a file', done => {
  createSourceFile$(`${__dirname}/index.ts`).subscribe(x => {
    expect(ts.isSourceFile(x.file)).toBeTruthy();
    done();
  });
});
