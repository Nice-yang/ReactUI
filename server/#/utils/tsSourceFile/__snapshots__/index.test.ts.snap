// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`non existing file 1`] = `
Object {
  "data": Object {
    "error": "{\\"errno\\":-2,\\"code\\":\\"ENOENT\\",\\"syscall\\":\\"open\\",\\"path\\":\\"\\"}",
    "fileName": "",
  },
  "level": 0,
  "message": "Cannot read file",
  "stack": undefined,
}
`;
